runtime/rococo/src/lib.rs

```diff
@@ -636,16 +687,9 @@ impl EnsureOrigin<Origin> for PriviledgedOrigin {
 	fn successful_origin() -> Origin { Origin::root() }
 }
 
-parameter_types! {
-	pub const ProposeDeposit: Balance = 1000 * DOLLARS;
-	pub const MaxNameLength: u32 = 20;
-}
-
-impl propose_parachain::Config for Runtime {
+impl validator_manager::Config for Runtime {
 	type Event = Event;
-	type MaxNameLength = MaxNameLength;
-	type ProposeDeposit = ProposeDeposit;
-	type PriviledgedOrigin = EnsureOneOf<AccountId, EnsureRoot<AccountId>, PriviledgedOrigin>;
+	type PrivilegedOrigin = EnsureOneOf<AccountId, EnsureRoot<AccountId>, PrivilegedOrigin>;
```

Comment:
```suggestion
	type PrivilegedOrigin =  EnsureRoot<AccountId>;
```

This is some functionality we only require root for. You could also remove this type completly and just call `ensure_root`. As this is mainly test code, I don't care what you actually do.